src/components/EvComponents/SponsorImage.jsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from "@mui/system";
+import React from "react";
+
+const SponsorImage = (props) => {
+    return (
+        <Box sx={{
+            textAlign: "center",
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            margin: "10px"
+        }}>
+            <img src={props.imageSrc}
```

Comment:
don't use vanilla img .. use the image compoennt from 
import Image from "components/BazarImage";

Reply:
solved